resources/views/WishList/WishListView.twig

```diff
@@ -7,7 +7,7 @@
 {% block PageBody %}
     {{ component( "Ceres::WishList.Components.WishList" ) }}
 
-    <div class="page-content basket container-max mb-5">
+    <div class="page-content wishlist container-max mb-5">
```

Comment:
.wishlist dann bitte noch in den Selektor aufnehmen in [_common.scss](https://github.com/plentymarkets/plugin-ceres/blob/c15e0afaf98b2d2d3983a1c6c26aad66c606fe24/resources/scss/ceres/common/_common.scss#L18)